nx-dev/ui-markdoc/src/lib/tags/stackblitz-button.schema.ts

```diff
@@ -0,0 +1,13 @@
+import { Schema } from '@markdoc/markdoc';
+
+export const stackblitzButton: Schema = {
+  render: 'StackblitzButton',
+  description: 'Renders a button to open a given repository in Stackblitz',
+  attributes: {
```

Comment:
The GitHub button title is an edge case as well. I added it because it looked better in some cases.

For the Stackblitz button I don't think we need a title 🤔. In case we can add it once we need it the first time